pkg/controllers/user/logging/generator/cluster_template.go

```diff
@@ -76,11 +76,11 @@ var ClusterTemplate = `{{ if .clusterTarget.CurrentTarget }}
     {{ if eq .clusterTarget.CurrentTarget "elasticsearch"}}
     @type elasticsearch
     include_tag_key  true
-    {{ if and .clusterTarget.ElasticsearchConfig.AuthUserName .clusterTarget.ElasticsearchConfig.AuthPassword}}
-    hosts {{.clusterTarget.WrapElasticsearch.Scheme}}://{{.clusterTarget.ElasticsearchConfig.AuthUserName}}:{{.clusterTarget.ElasticsearchConfig.AuthPassword}}@{{.clusterTarget.WrapElasticsearch.Host}}
-    {{else -}}
+    {{- if and .clusterTarget.ElasticsearchConfig.AuthUserName .clusterTarget.ElasticsearchConfig.AuthPassword}}
+    user {{.clusterTarget.ElasticsearchConfig.AuthUserName}}
+    password {{.clusterTarget.ElasticsearchConfig.AuthPassword}}
+    {{- end }}
     hosts {{.clusterTarget.ElasticsearchConfig.Endpoint}}    
```

Comment:
Directly get all different from 2.2 will cause some problem, because of 2.2 include some feature that 2.1 doesn't have, like Kafka sasl, some fields not in 2.1 will cause some problem, like ```SaslUsername```, ```SaslUsername```. Separate user and password from hosts for cluster and project template could fix this issue.